lib/govuk_app_config/govuk_json_logging/rails_ext/action_controller/metal/instrumentation.rb

```diff
@@ -0,0 +1,56 @@
+module ActionController
+  module Instrumentation
+    alias_method "orig_process_action", "process_action"
+    def process_action(*args)
```

Comment:
Is this copy-pasted from anywhere? If so, please add the link - and if not, this might be one of those rare occasions that actually benefits from interspersing more inline comments explaining what you're doing in each sequence of steps 🙏

Reply:
Added comments to highlight whats changed. The only change is updating the raw_payload to be similar to whats used in Rails 7.1.

Reply:
Ah ok - thanks 👍